3rdparty/jvm/com/fasterxml/jackson/module/BUILD

```diff
@@ -0,0 +1,6 @@
+jar_library(
```

Comment:
nit: Copyright header, if we're adding them, like in  3rdparty/jvm/org/scala-lang/modules/BUILD